packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -93,7 +95,35 @@ - (void)alertMessage:(NSString *)message
                                                                                    completion:nil];
 }
 
-- (void)getAvailableBiometrics:(FlutterResult)result {
+- (void)deviceSupportsBiometrics:(FlutterResult)result {
+  LAContext *context = self.createAuthContext;
+  NSError *authError = nil;
+  // Check if authentication with biometrics is possible.
+  if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
+                           error:&authError]) {
+    if (authError == nil) {
+      result(@YES);
+      return;
+    }
+  }
+  // If not, check if it is because no biometrics are enrolled (but still present).
+  if (authError != nil) {
+    if (@available(iOS 11.0.1, *)) {
+      if (authError.code == LAErrorBiometryNotEnrolled) {
+        result(@YES);
+        return;
+      }
+    }
+    if (authError.code == LAErrorTouchIDNotEnrolled) {
+      result(@YES);
+      return;
+    }
+  }
+
```

Comment:
is there a corresponding `LAErrorFaceIDNotEnrolled` error that needs to be handled?

Reply:
No, `LAErrorBiometryNotEnrolled` is the result of them adding a second biometric type and realizing the scaling problem they had with their initial naming, and fixing it at that point. See [my new comment](https://github.com/flutter/plugins/pull/5329#discussion_r856495027) :)